shared/chat/inbox/row/chat-filter-row/index.tsx

```diff
@@ -51,6 +52,13 @@ class ConversationFilterInput extends React.PureComponent<Props> {
     }
   }
 
+  private hotKeys = Platforms.isDarwin ? ['command+n'] : ['ctrl+n']
+  private onHotKeys = (cmd: string) => {
+    if (cmd.endsWith('+n')) {
```

Comment:
why is this necessary if the only hotkey ends with +n? What else would get passed here?